src/admin/shared/components/FilterTable/FilterTable.utils.ts

```diff
@@ -0,0 +1,13 @@
+import { isArray, isEmpty, isNil, isPlainObject, isString, omitBy } from 'lodash';
```

Comment:
prefer lodash-es, it's better tree shakeable
https://til.hashrocket.com/posts/dnzttruljf-prefer-lodash-es-when-using-webpack

Reply:
https://meemoo.atlassian.net/browse/AVO-1947